core/roboconf-target-docker/src/main/java/net/roboconf/target/docker/internal/DockerHandler.java

```diff
@@ -148,7 +148,7 @@ else if(( image = DockerUtils.findImageByTag( imageId, images )) != null )
 			.createContainerCmd(imageId)
 			.withCmd("/usr/local/roboconf-agent/start.sh",
 						"application-name=" + applicationName,
-						"root-instance-name=" + rootInstanceName,
+						"scoped-instance-path=" + scopedInstancePath,
```

Comment:
I am not sure. At least, I have not seen anything in the code.
The Docker file generator reads all the properties and injects them in the agent's configuration file.
However, if you had existing images, you will have to remove/update them to work with a new DM.